README.md

```diff
@@ -8,90 +8,132 @@ We're migrating from using Jekyll to using
 [mdBook](https://github.com/rust-lang/mdBook). The mdBook version is located in
 the `./better-code` directory and includes automated CI/CD deployment to GitHub Pages.
 
-## Installing and updating mdBook
+The published book is available at: https://stlab.github.io/better-code/
 
-Install [Rust and
-Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html):
+## Prerequisites
 
-Linux and macOS:
+Install [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html):
 
-```
+**Linux and macOS:**
+```bash
 curl https://sh.rustup.rs -sSf | sh
 ```
 
-On Windows, you can download the installer from
-[here](https://win.rustup.rs/).
+**Windows:**
+Download the installer from [here](https://win.rustup.rs/).
 
-Once you have Rust and Cargo installed, you can install or upgrade mdBook by running:
+## Installing mdBook and Plugins
 
-```
-cargo install mdbook
+**Important**: To ensure consistency between local development and CI, all tool
+versions are centrally managed in `versions.toml`. Use the provided installation
+scripts to automatically install the correct versions:
+
+**Linux/macOS:**
+```bash
+./scripts/install-tools.sh
 ```
 
-## Building the book
+**Windows (PowerShell):**
+```powershell
+.\scripts\install-tools.ps1
+```
 
-To build the book, run:
+These scripts automatically install mdBook and all required plugins using the
+versions specified in `versions.toml`. This is the same file used by CI, ensuring
+perfect consistency.
 
+**Manual Installation:**
+If you prefer to install manually, check `versions.toml` for the current version
+numbers, then run:
+```bash
+cargo install mdbook --version <version-from-toml>
+cargo install mdbook-katex --version <version-from-toml>
```

Comment:
If this is sufficient then  only two version numbers need to be in `versions.toml` and the whole complex scheme is massive overkill.  And in fact there are only two versions there.  I'm 100% sure even if repeating two versions numbers must be avoided, there's a simpler way to do this.  Like, give different manual installation commands for Windows and Linux that use the contents of 2 separate files, or the first and second line of one file, or that grab the two versions from one JSON file.  And then the installation scripts are totally unneeded.

Reply:
Except the two versions need to be consistent and match locally and in CI - I didn't want to have to keep updating versions in multiple places. We are likely to add more dependencies for diagramming, Swift examples, and so on.